packages/open-payments/src/test/helpers.ts

```diff
@@ -1,8 +1,53 @@
 import createLogger from 'pino'
 import { createAxiosInstance } from '../client/requests'
+import { ILPStreamConnection, IncomingPayment } from '../types'
+import base64url from 'base64url'
+import { v4 as uuid } from 'uuid'
```

Comment:
just for some more "believable" test data, doesn't add much at this point, but decided to have it